docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+services:
+    app:
+        build:
+            context: .
+        ports:
+            - "8000:8000"
+        volumes:
+            - ./:/app/
+            - my_media:/files/media
+            - my_static:/files/static
+        command: >
+            sh -c "python manage.py makemigrations &&
+            python manage.py wait_for_db
```

Comment:
The `wait_for_db` command is correctly included to ensure the database is ready before running migrations. Ensure that the `wait_for_db.py` script is correctly implemented to handle database connection retries.